doc/modules/clustering.rst

```diff
@@ -278,6 +285,67 @@ small, as shown in the example and cited reference.
    D. Sculley, *Proceedings of the 19th international conference on World
    wide web* (2010)
 
+.. _k_medoids:
+
+K-Medoids
+=========
+
+
+:class:`KMedoids` is related to the :class:`KMeans` algorithm. While
+:class:`KMeans` tries to minimize the within cluster sum-of-squares,
+:class:`KMedoids` tries to minimize the sum-of-distances between points labeled
+as the medoids of the cluster an all points assigned to that cluster.
+Furthermore, the distance metric used by :class:`KMedoids` can be freely chosen
```

Comment:
I think the causal relationship should be clearer: "The use of a data point to represent each cluster's center allows the use of any distance metric for clustering."